src/bidiMapper/modules/context/BrowsingContextImpl.ts

```diff
@@ -1974,6 +1974,14 @@ export class BrowsingContextImpl {
     );
   }
 
+  async setTouchOverride(maxTouchPoints: number | null) {
+    await Promise.all(
```

Comment:
I think we should do await Promise.allSettled here since for some targets the command will error and we probably do not want to return early.